src/Game/ShipSpawner.cpp

```diff
@@ -168,6 +245,24 @@ bool ShipSpawner::is_spawn_possible() const
     return false;
 }
 
+std::string ShipSpawner::spawn_type_to_string(SpawnType const type)
+{
+    switch (type)
+    {
+    case SpawnType::Sequence:
+        return "Sequence";
+
+    case SpawnType::Imidiet:
```

Comment:
Immediate instead of imidiet.

Reply:
I think i found and replace every occurence of this misspelled